sys/newlib/syscalls.c

```diff
@@ -71,8 +72,8 @@ void _fini(void)
  */
 void _exit(int n)
 {
-    printf("#! exit %i: resetting\n", n);
-    reboot(n);
+    LOG_INFO("#! exit %i: resetting\n", n);
```

Comment:
Probably. That probably also was the case before on some platforms. We could xtimer_wait() this out, but I'd then rather do it only with DEVELHELP. What do you think?